client/src/index.test.js

```diff
@@ -34,16 +34,4 @@ describe('wagtail package API', () => {
   it('has Transition', () => {
     expect(Transition).toBeDefined();
   });
-
```

Comment:
`Explorer` `ExplorerToggle` and `initExplorer` imports are not needed anymore in this file

Reply:
Oops, fixed!